EMS/core-bundle/src/Repository/WysiwygProfileRepository.php

```diff
@@ -20,50 +22,17 @@ public function __construct(Registry $registry)
         parent::__construct($registry, WysiwygProfile::class);
     }
 
-    /**
-     * @return WysiwygProfile[]
-     */
-    public function findAll(): array
```

Comment:
Why deleting it here and adding it in the wyswyygStyleSetProfile? Doesn't some default order is defined in a parent class?

Reply:
It was only used in the service (which I also removed)

```php
/**
     * @return WysiwygProfile[]
     */
    public function getProfiles(): array
    {
        $profiles = $this->wysiwygProfileRepository->findAll();

        return $profiles;
    }
```

And the getProfiles methods was only used in the WysiwygController:

**newAction** 
>setOrderKey(100 + \count($this->wysiwygProfileService->getProfiles())); 
used the count method on the service

**indexAction**
> 'profiles' => $this->wysiwygProfileService->getProfiles(), 
removed not needed we have a datatable

The selection in the forms for profilles:

```php
->add('wysiwygProfile', EntityType::class, [
                'required' => false,
                'label' => 'WYSIWYG profile',
                'class' => WysiwygProfile::class,
                'choice_label' => 'name',
                'query_builder' => fn (EntityRepository $er) => $er->createQueryBuilder('p')->orderBy('p.orderKey', 'ASC'),
                'attr' => [
                    'data-live-search' => true,
                    'class' => 'wysiwyg-profile-picker',
                ],
            ])
```

Reply:
And for the styleSets the findAll is still used in the service

```php
/**
     * @return WysiwygStylesSet[]
     */
    public function getStylesSets(): array
    {
        static $stylesSets = null;

        return $stylesSets ?? $this->wysiwygStylesSetRepository->findAll();
    }
````

Reply:
Indeed, the findAll is never used by the DataTable mechanisms... sorry, it was silly